### PR TITLE
DBZ-4891 Oracle Logminer : escaped single quote handling in DML

### PR DESCRIPTION
**Pull request by AleksejsSibanovs**
Fix Oracle Logminer's escaped single quotes parsing in DMLs, related unit test and fix for the old DBZ-3413 test

### CODE REVIEW

**Comment by AleksejsSibanovs** [comment]
Looks like locally I just run the style reformatting, which doesn't case about StringBuffer. Investigated, understood, changed to StringBuilder

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlParser.java [hunk 1]
```diff
@@ -276,8 +276,24 @@ private int parseColumnValuesClause(String sql, int start, String[] columnNames,
         index += VALUES_LENGTH;
 
         int columnIndex = 0;
+        StringBuilder collectedValue = null;
         for (; index < sql.length(); ++index) {
+            char lookAhead = (index + 1 < sql.length()) ? sql.charAt(index + 1) : 0;
```
Comment (Naros) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlParser.java:281]: This is used, is this intended?

**Comment by Naros** [comment]
Hi @AleksejsSibanovs I pushed one commit here to address a similar concern with `SEL_LOB_LOCATOR` events.  I'm going to remove the unused `lookAhead` in that one method so we can wrap up this PR in preparation for this week's release.

**Comment by Naros** [comment]
Reworked commit messages, squashed my commits & applied to `main`.
Thanks @AleksejsSibanovs !!!

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlParser.java (hunk 1 above)
Comment (AleksejsSibanovs) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlParser.java]: My fault, looks like this one introduced in some intermediary version and later somehow survived the refactoring.

**Comment by quydx** [comment]
Has this bug been fixed on version 1.9.0 CR1, I can not find version 1.9 final in download page ?

**Pull request closed without merging**